front_end/src/App.css

```diff
@@ -386,3 +386,89 @@ a:active {
   top: 23px;
   position: relative;
 }
```

Comment:
Please create your own .css file, if possible in the same folder as your .js stuff.
don't forget to import it.